redfish-core/lib/account_service.hpp

```diff
@@ -2001,6 +2026,28 @@ inline void requestAccountServiceRoutes(App& app)
                 }
             }
 
+
+            // Don't PATCH any account which has a Restricted Role,
+            // except allow the Locked property.
+            // Do not even allow the service user to change these properties.
+            // Implementation note: Ideally this would get the user's RoleId
+            // but that would take an additional D-Bus operation.
+            if ((username == "service") &&
```

Comment:
We went with https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48633/4/redfish-core/lib/account_service.hpp
over 
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48564

Can the code be wrote more like https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48633

Reply:
Yes, I will make that change.